src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -179,15 +179,50 @@ export default {
 			mdiFilterVariant,
 			selectedCategory: 0,
 			selectedGender: '',
-			sortBy: 'personalized',
-			showBadge: false
+			sortBy: 'amountLeft',
+			showBadge: false,
+			presetFilterActive: {
+				women: false,
+				kivaUs: false,
+				endingSoon: false,
+			},
 		};
 	},
 	mixins: [
 		loanChannelQueryMapMixin
 	],
 	watch: {
-		selectedCategory() {
+		selectedCategory(categoryId) {
+			const catId = Number(categoryId);
+			const queryMap = loanChannelQueryMapMixin.data().loanChannelQueryMap;
+			const categoryFilter = catId === 0 ? {} : queryMap
+				.find(channel => channel.id === catId)?.flssLoanSearch;
+
+			if (this.presetFilterActive.women) {
```

Comment:
Couldn't all filters just be reset instead when changing category?

Reply:
design/product wants just the preset ones to be reset for the "special" categories!